catroid/src/main/java/org/catrobat/catroid/devices/mindstorms/nxt/sensors/NXTI2CUltraSonicSensor.java

```diff
@@ -38,8 +38,8 @@
 	private static final int DEFAULT_VALUE = 255;
 	private static final String TAG = NXTI2CUltraSonicSensor.class.getSimpleName();
 
-	private enum DistanceUnit {
-		DUMMY, CENTIMETER, INCH;
```

Comment:
PLEASE EXPLAIN: Why did you move this down?

Reply:
I think that when I checked the refactor and reformat code option. it changed the place of the method 
![image](https://user-images.githubusercontent.com/23483020/76852696-8c621080-6854-11ea-9295-f70cb07f87a6.png)
I will move it now. Thanks